perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -366,8 +508,17 @@ def Run(benchmark_spec):
   Returns:
     A list of sample.Sample objects.
   """
-  RunCassandraStressTest(benchmark_spec)
-  return CollectResults(benchmark_spec)
+  metadata = SetupMetadata(benchmark_spec)
+  RunCassandraStressTest(
+      benchmark_spec.vm_groups[CASSANDRA_GROUP],
+      benchmark_spec.vm_groups[CLIENT_GROUP],
+      metadata['num_keys'],
+      metadata.get('command'),
+      metadata.get('ops'),
+      metadata.get('pop_size'),
```

Comment:
At least `command` and `pop_*` appear to always occur in `metadata`. Why `.get` for those and not for `num_keys`?